.github/workflows/test.yml

```
@@ -0,0 +1,223 @@
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # Linux x86_64
+          - target: x86_64-unknown-linux-gnu
+            runner: ubuntu-22.04
+            features: ""
+          - target: x86_64-unknown-linux-gnu
+            runner: ubuntu-22.04
+            features: "jemalloc"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)